examples/cryptocurrency-advanced/backend/build.rs

```diff
@@ -1,10 +1,13 @@
-use exonum_build::{get_exonum_protobuf_files_path, protobuf_generate};
+use exonum_build::{protobuf_generate, ProtoSources};
 
 fn main() {
-    let exonum_protos = get_exonum_protobuf_files_path();
     protobuf_generate(
         "src/proto",
-        &["src/proto", &exonum_protos],
+        &[
+            ProtoSources::Exonum,
```

Comment:
I think that this combination of proto files is very commonly used and then you should provide a single method to get them together.